dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -404,7 +404,12 @@ def SetUp(self,
       self.extra_artifacts = [item.strip() for item
                               in extra_artifacts.strip().split(',')]
 
-    self.hostnames = [item.strip() for item in hosts.strip().split(',')]
+    for item in hosts.strip().split(','):
```

Comment:
How about renaming "hosts" here to "hostnames"? This is a list of hostnames, not host containers